prog/detect/sensors-detect

```diff
@@ -6364,6 +6369,33 @@ sub fsc_detect
 	return 8;
 }
 
+# Chip to detect: FSC Teutates
+# Registers used:
+#   0x00: Device ID
+#   0x01: Revision
+#   0x0C: Device Detection Register 1
+#   0x0D: Device Detection Register 2
+#   0x0E: Device Detection Register 3
+sub fts_detect
+{
+	my ($file) = @_;
+
+	# Device ID must be 0x11
+	# 0x10 == Baseboard Management Controller
+	# 0x01 == Teutates
+	return unless i2c_smbus_read_byte_data($file, 0x00) == 0x11;
+
+	# Revision must be greater or equal 0x2b to use device detection
+	return unless i2c_smbus_read_byte_data($file, 0x01) >= 0x2b;
+
+	# Device Detection Registers must be 0x17, 0x34, 0x54
+	return unless i2c_smbus_read_byte_data($file, 0x0C) == 0x17
+		  and i2c_smbus_read_byte_data($file, 0x0D) == 0x34
+		  and i2c_smbus_read_byte_data($file, 0x0E) == 0x54;
```

Comment:
That's an odd mixture of tabs and spaces. Use tabs for indentation, spaces for alignment. I.e. one tab, the rest should be spaces.